src/components/Footer/Footer.module.css

```diff
@@ -1,9 +1,112 @@
 .footer {
-    background-color: #f8f9fa;
-    padding: 10px 20px;
-    box-shadow: 0 -2px 4px rgba(0, 0, 0, 0.1);
-    width: 100%;
-    bottom: 0;
-    left: 0;
-    text-align: center;
-  }
\ No newline at end of file
+  max-width: 100%;
+  background-color: var(--dark-green);
+  color: var(--primary-white);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-top: 66px;
+  padding-bottom: 64px;
+  padding-inline: 24px;
+  box-sizing: border-box;
+}
+
+.footerInfo {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.footerLinks {
+  display: flex;
+  justify-content: center;
+  gap: 1.5rem;
+  transition: all 3ms ease-in;
```

Comment:
Not sure what this transition is accomplishing? At 3ms, it's not animating anything that the human eye can appreciate.